queue_services/account-mailer/src/account_mailer/email_templates/common/style.html

```diff
@@ -4,8 +4,8 @@
    */
   @font-face {
     font-family: 'BCSans-Regular';
-      src: url('https://minio-dev.pathfinder.gov.bc.ca/public/email_template_font_woff2_style.woff2') format('woff2'),
-          url('https://minio-dev.pathfinder.gov.bc.ca/public/email_template_font_woff_style.woff') format('woff');
+      src: url('https://registries-minio.pathfinder.gov.bc.ca/public/BCSans-Regular.woff2') format('woff2'),
```

Comment:
Is it possible to use the minio url  and bucket value from config? In case we need change it after openshift 4 migration.